custom/enikshay/ucr/data_sources/episode_v3_1.json

```diff
@@ -16,22 +16,20 @@
         "description": "",
         "base_item_expression": {
         },
-        "table_id": "episode_v3",
-        "display_name": "Episode v3",
+        "table_id": "episode_v3_1",
+        "display_name": "Episode v3.1",
         "configured_filter": {
             "type": "and",
-            "filters": [
-                {
-                    "operator": "eq",
-                    "expression": {
-                        "datatype": null,
-                        "type": "property_name",
-                        "property_name": "type"
-                    },
-                    "type": "boolean_expression",
-                    "property_value": "episode"
-                }
-            ]
+            "filters": {
```

Comment:
Flagging that you changed `filters` from a list to a dict, but this is an `and` filter.  Does that work?

Reply:
(saw that you resolved this later)